linux_os/guide/system/network/network-nftables/nftables_rules_permanent/tests/invalid_top_level_config_file.fail.sh

```diff
@@ -0,0 +1,3 @@
+# platform = sle15
+
+echo 'include /some/invalid/path' > {{{ var_nftable_master_config_file }}}
```

Comment:
You can't use variables in test scenarios this way, it won't work. You need to use a specific value in your code. Then set the variable to that value in the scenarios header, or you can define a specific profile. 

```
# variables = var_nftable_master_config_file=/etc/sysconfig/nftables.conf

echo 'include /some/invalid/path' > "/etc/sysconfig/nftables.conf"
```

Reply:
:+1: should be ok in 4f3064e